### DISCUSSION TITLE
Visual Elements

### DISCUSSION THREAD

**Discussion by Bumblebee001** [GrandOrgue/GoOdf#25] [General]
Where can one see the graphics to be able to choose and select?

**Comment by larspalo** [comment]
Which graphics? The DispImagNum usually are a wxBitmapComboBox that shows what the value would be. The same is for the different display metrics backgrounds.

Actual graphical rendering of the panels exactly like GO does, with the possibility to re-arrange the positioning at will, is not implemented yet, but it's on my todo-list of desired features!

**Reply by Bumblebee001** [reply]
Eagerly looking forward to that.... I feel that this is very important and should be given some priority. I have been creating sample-sets, like you, for many years now as you know. I hope some features are introduced soon.
